RELEASENOTES.md

```diff
@@ -33,6 +33,12 @@
 > [!NOTE]
 > The projects mentioned here are AL-Go projects in a multi-project repository. A repository can contain multiple projects and a project can contain multiple apps.
 
+### Run "Update AL-Go System Files" on a schedule on multiple branches
+
+When run on a schedule, _Update AL-Go System Files_ only runs on the _main_ branch. By setting `includeBranches` in `workflowSchedule` setting, you can now run the workflow on a schedule on multiple branches. Read more at https://aka.ms/algosettings#workflowSchedule.
```

Comment:
Please start this by describing a new input parameter to Update AL-Go System Files called includeBranches and what it does - and then add that it can be set on workflowSchedule

Reply:
Changed the release notes. Please review again.